Created: project

katleon opened the pull request:
Usunęłam pliki z example oraz container.scss, main.css i main.css.map; 
Przeniosłam plik reset.scss do folderu "base";
Utworzyłam plik common.scss;
Dodałam fonty do pliku variabes.scss;
Załączyłam plik products.scss do pliku main.scss;
Załączyłam plik products.hmtl do pliku index.html;

katleon committed:
Created: project

Usunęłam pliki z example oraz container.scss, main.css i main.css.map;
Przeniosłam plik reset.scss do folderu "base";
Utworzyłam plik common.scss;
Dodałam fonty do pliku variabes.scss;
Załączyłam plik products.scss do pliku main.scss;
Załączyłam plik products.hmtl do pliku index.html;

Wiecek-K reviewed (commented):
do sprawdzenia przez większą ilość osób zmiany w stylach globalnych/klasach z których większość korzysta

Wiecek-K reviewed (commented):
dajcie mi też info czy widzicie że jakieś pliki oznaczyłem jako "viewed"
jeśli te flagi są widoczne tylko dla mnie to do pliki w których występują zmiany globalne to 
src/sass/base/_reset.scss
src/sass/base/_common.scss
src/sass/components/_container.scss
tak jak pisałem na slacku jak druga osoba kliknie approve i nie będzie żadnego sprzeciwu to niech ta druga robi od razu merge żeby każdy mógł swoje branche zaktualizować

karollaski commented:
Wiecek-K niestety nie widzę Twoich oznaczenń "viewed", no chyba, że jakoś źle patrzę 😅

katleon committed:
Changed: container.scss and font inherit

Usunęłam common.scss z folderu base;
Zmieniłam font na dziedziczny z body;
Usunęłam font 'Titan One' z variables.scss;
Przeniosłam products.scss do folderu layouts;
Przywróciłam style dziedziczne z html w pliku reset.scss,
